tests/test_scope.py

```diff
@@ -25,6 +25,10 @@ def test_copying():
     s1.set_tag("foo", "bar")
 
     s2 = copy.copy(s1)
+    # Check all attributes are copied
+    for name in set(Scope.__slots__) - {"client"}:
+        getattr(s2, name)
+
```

Comment:
I would prefer to add this check to a separate unit test, since it is somewhat unrelated to the tag copying checks that this existing test is checking. The comment should also be updated to something like "Check that all attributes, except `client`, are present on the new scope", as the check only verifies the attributes' presence, and does not check whether the values were actually copied.